test/runner/lib/sanity/__init__.py

```diff
@@ -145,8 +148,13 @@ def collect_code_smell_tests():
     :rtype: tuple[SanityFunc]
     """
     skip_file = 'test/sanity/code-smell/skip.txt'
+    ansible_only_file = os.path.join(INSTALL_ROOT, 'test/sanity/code-smell/ansible-only.txt')
```

Comment:
What does this file signify?

Reply:
This file lists the code-smell sanity tests which are currently executed only on Ansible and not collections. Some of these may updated to work on collections later.